### DISCUSSION TITLE
v1.3.1 pre-release:  More support for SC Ivo files.  Hunt fix

### DISCUSSION THREAD

**Discussion by Markemp** [Markemp/Cryengine-Converter#78] [Announcements]
Currently in pre-release.

* Fix for Hunt issue on 1.3.0 release
* Fix bad quaternion to rotation matrix calculation
* Add support for 2 new Normals datastream Ivo chunks

A discussion has been opened for this pre-release.  Please comment there to help identify models that were fixed as a release, and new models that stopped working, as well as general comments.

<hr /><em>This discussion was created from the release <a href='https://github.com/Markemp/Cryengine-Converter/releases/tag/v1.3.1'>More support for SC Ivo files.  Hunt fix</a>.</em>

**Comment by lonelyhawk** [comment]
could you please add more details here? A sample file, maybe?

**Reply by Markemp** [reply]
The Hunt issue was due to a conflict with SC on the CrChF files.  I found a property in the model that is helping me differentiate between the two games so that both will use the correct logic for each.

I'm not sure what the quaternion issue is anymore.  I let this release fester too long. :(

The #ivo file was just to add more datastream support.  In hindsight I don't think they are all normal files; one might be colors2.  Still a lot of unknowns with that file format, unfortunately.

I'm going to spend the weekend working on getting this verified and released.  1.3.1 seems pretty stable.

**Reply by Axehandler** [reply]
> I'm going to spend the weekend working on getting this verified and released. 1.3.1 seems pretty stable.

I'm curious if you could add a time out feature so it goes to the next item or ends.....

when processing SC skins.. it will just Hang on different skins,

--example.,

D:> D:\SC\Data\Objects\cgf-converter.new.exe D:\SC\Data\Objects\fps_weapons\weapons_v7\gmni\sniper\a03\bsnp_fps_gmni_a03_parts.skin -noconflict
Input file set to D:\SC\Data\Objects\fps_weapons\weapons_v7\gmni\sniper\a03\bsnp_fps_gmni_a03_parts.skin
Prevent conflicts for mtl files enabled

this will sit here all night if I let it.

Makes it tough to re-curse the directory's and process a bunch of stuff.

THANK YOU for your consideration and REALLY apricate all the work! 👯
Axe

**Reply by Kjasi** [reply]
> when processing SC skins.. it will just Hang on different skins,

Actually, I fixed that issue. There just hasn't been an official build (or beta build) since then...

**Reply by Axehandler** [reply]
> > when processing SC skins.. it will just Hang on different skins,
> 
> Actually, I fixed that issue. There just hasn't been an official build (or beta build) since then...

I was SOOO looking forward to this...

however with the previous version 1.3.0 the kabuki mask (skin) would come out fine.
but the new version 1.3.1 the kubuki mask (skin) same file doesn't appear to have geometry.

D:\SC\Data\Objects\Characters\Human\male_v7\armor\outlaw\kabuki_mask_01_out.dae
is the file loaded on both. 

notice the 2nd one doesn't even show the file - it's only showing "Armature.001" 

I created 2 copies of the folder and processed each.
this appears to be happening consistently with multiple Skin and CGF files.


See the 2 attached files
![kubuki 1 3 0](https://user-images.githubusercontent.com/91401793/135696059-68541acb-ca15-4839-8e6f-1f04c1aad5ad.jpg)
![kubuki 1 3 1](https://user-images.githubusercontent.com/91401793/135696060-9b5f1965-4a8e-4606-87ca-f9aadabb1ce1.jpg)
.
[Uploading Diffrence dae and skins.zip…]()
